src/core/portablemc/cli.py

```diff
@@ -499,10 +499,19 @@ def _predicate(spec: "LibrarySpecifier") -> bool:
                     }, done=True)
 
         if ns.jvm is None:
-            print_task("", "start.jvm.loading")
-            version.prepare_jvm()
-            print_task("OK", "start.jvm.loaded", {"version": version.jvm_version}, done=True)
+            try:
+                print_task("", "start.jvm.loading")
+                version.prepare_jvm()
+                print_task("OK", "start.jvm.loaded", {"version": version.jvm_version}, done=True)
+            except JvmLoadingError:
+                ns.jvm = shutil.which(get_jvm_bin_filename())
+                if ns.jvm is not None:
+                    print_task("OK", "start.jvm.system_fallback", {"path": ns.jvm})
```

Comment:
Needs `, done=True` to avoid getting overwritten by next `print_task()` call I believe?

Reply:
right!